react/src/components/atoms/buttons/ButtonAlert/ButtonAlert.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import ButtonAlert from './index';
+import ButtonAlertDocs from './ButtonAlert.md';
+
+storiesOf('atoms/buttons', module)
+  .addDecorator(withKnobs({ escapeHTML: false }))
+  .add(
+    'ButtonAlert', (() => {
+      const props = {
+        onClick: action('ButtonAlert clicked'),
+        text: text('text', ' Updates'),
+        showText: text('showText', 'Show'),
+        hideText: text('hideText', 'Hide')
```

Comment:
Lets also add the new classes prop:
```suggestion
        hideText: text('hideText', 'Hide'),
        classes: text('classes', ''),
```